src/Symfony/Component/Mime/MimeTypes.php

```diff
@@ -2377,20 +2568,25 @@ public function guessMimeType(string $path): ?string
         'msf' => ['application/vnd.epson.msf'],
         'msg' => ['application/vnd.ms-outlook'],
         'msh' => ['model/mesh'],
-        'msi' => ['application/x-msdownload', 'application/x-msi'],
+        'msi' => ['application/octet-stream', 'application/x-msdownload', 'application/x-msi'],
```

Comment:
I've checked the source and I think we should always list `'application/octet-stream'` last.
Alternatively, we should just skip it. The reason is that we already fallback to `'application/octet-stream'` by default in `DataPart`. For suren listing it first is going to be a regression.

Reply:
@nicolas-grekas you prefer that:

a) I put 'application/octet-stream' last.
b) I skip 'application/octet-stream'

Reply:
Skipping would be best IMHO